src/databricks/labs/ucx/contexts/workspace_cli.py

```diff
@@ -179,6 +180,10 @@ def iam_role_creation(self):
     def notebook_loader(self) -> NotebookLoader:
         return NotebookLoader()
 
+    @cached_property
+    def history_installation(self) -> HistoryInstallation:
```

Comment:
not sure if it's the best name.

Reply:
Me neither, I copied the "Installation" part from other objects, like `WorkspaceInstallation` or `AccountInstallation`, then added `run` method to run the installation.

I actually prefer the word "artifacts", which is not very descriptive and some of the objects in the history table are not the "byproduct" of UCX but the "main product". On the other hand, it describes what we are doing here: installing the resources for UCX artifacts (like scan runs, errors and the history table).

What do you think?